### PR TITLE
translate 1-if-zero-string to fa

### PR DESCRIPTION
**Pull request by ali-sefidmouy**
This is my first PR on this repo! Please read the commit's comment to help me if you could. Thanks!

### CODE REVIEW

File: 1-js/02-first-steps/10-ifelse/1-if-zero-string/task.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-importance: 5
+اهمیت: 5
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/10-ifelse/1-if-zero-string/task.md:1]: As mentioned [here](https://github.com/javascript-tutorial/fa.javascript.info#metadata), do not translate meta words like `importance`.

**Comment by mehradsadeghi** [comment]
> I have a problem with the RTL format. Would be nice if you could help.

Please take a look at comments in this [PR](https://github.com/javascript-tutorial/fa.javascript.info/pull/57#issuecomment-653320058). If it didn't resolve your issue, describe the problem here.

File: 1-js/02-first-steps/10-ifelse/1-if-zero-string/task.md (hunk 1 above)
Comment (ali-sefidmouy) [reply 1-js/02-first-steps/10-ifelse/1-if-zero-string/task.md:1]: Yeah, thanks for your hint.
I mean a line that composed of two languages, such as the 7'th line of task.md file.

**Comment by mehradsadeghi** [comment]
> > I have a problem with the RTL format. Would be nice if you could help.
> 
> Please take a look at comments in this [PR](https://github.com/javascript-tutorial/fa.javascript.info/pull/57#issuecomment-653320058). If it didn't resolve your issue, describe the problem here.

@ali-sefidmouy 

Yeah. Your issue is exactly what I discussed with @newshahn before in above link.

File: 1-js/02-first-steps/10-ifelse/1-if-zero-string/task.md (hunk 1 above)
Comment (kooshan75) [reply 1-js/02-first-steps/10-ifelse/1-if-zero-string/task.md:1]: > This is my first PR on this repo! Please read the commit's comment to help me if you could. Thanks!

Hey, have you replaced "اهمیت" to "importance"? If you have, please commit it in order to merge the request with your own name :)

**Pull request merged**